I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/main_menu.rpy", line 13, in script
    python hide:
  File "game/main_menu.rpy", line 13, in script
    python hide:
  File "game/main_menu.rpy", line 13, in <module>
    python hide:
  File "game/main_menu.rpy", line 19, in _execute_python_hide
    'Quit Game' : dict(xpos=550, ypos=490, xanchor='left', yanchor='top'),
Exception: config.main_menu_positions is not a known configuration variable.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 326, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 515, in main
    renpy.game.context().run(node)
  File "game/main_menu.rpy", line 13, in script
    python hide:
  File "game/main_menu.rpy", line 13, in script
    python hide:
  File "renpy/ast.py", line 913, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2111, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/main_menu.rpy", line 13, in <module>
    python hide:
  File "game/main_menu.rpy", line 19, in _execute_python_hide
    'Quit Game' : dict(xpos=550, ypos=490, xanchor='left', yanchor='top'),
  File "renpy/defaultstore.py", line 106, in __setattr__
    raise Exception('config.%s is not a known configuration variable.' % (name))
Exception: config.main_menu_positions is not a known configuration variable.

